src/components/safe-apps/AppFrame/useThirdPartyCookies.ts

```diff
@@ -0,0 +1,71 @@
+import { useState, useEffect, useRef, useCallback } from 'react'
+
+export const THIRD_PARTY_COOKIES_CHECK_URL = 'https://third-party-cookies-check.gnosis-safe.com'
```

Comment:
Need to update this.

Reply:
do you mean the devops team pointing to a new domain right?

Reply:
I would add `THIRD_PARTY_HOST` to `constants.ts` (with this old URL for now) so that we can update it in future easily.

Reply:
done